### DISCUSSION TITLE
Multiple hostnames under the `dns_name` field

### DISCUSSION THREAD

**Discussion by mochipon** [netbox-community/netbox#9232] [Ideas]
I would like to double-check the direction of this project regarding https://github.com/netbox-community/netbox/issues/4197. I have read the contributing guide and the issue intake policy, but please excuse me if this is an inappropriate question.

I am maintaining [sukiyaki/octodns-netbox](https://github.com/sukiyaki/octodns-netbox), which allows us to properly manage DNS records across multiple providers based on NetBox data. At first I used the `description` field as a place to store **comma-separated** FQDN**s**, but since NetBox now has a `dns_name` field, I have received several PRs (https://github.com/sukiyaki/octodns-netbox/pull/1, https://github.com/sukiyaki/octodns-netbox/pull/2) to take advantage of this.

By the discussion at https://github.com/netbox-community/netbox/issues/4197, I understand that any enhancements for storing multiple FQDNs in the `dns_field` field will not be allowed forever, is this correct?

I have found that a wildcard record has been implemented recently. This appears to be a deviation (extension) of the rule of "hold a single name only", at least conceptually. In addition, it appears that this enhancement, despite being passed over at one time https://github.com/netbox-community/netbox/issues/4973, was considered again at the suggestion of the community https://github.com/netbox-community/netbox/issues/8785. 

Therefore, I would like to know if there are any possibility of enhancements to the `dns_name` field to decide the direction of my project. Thank you!

**Comment by DanSheps** [comment]
The stance likely isn't changing.

That said, there is at least 1 plugin for managing DNS in netbox.
